test/fixtures/RunCommand/integration-lifecycle/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "independent",
+  "scripts": {
+    "postinstall": "lerna run my-script --scope=package-1"
+  }
```

Comment:
I really don't think we need a "lerna run" from npm script integration test, we're already doing that in `lerna-bootstrap.test.js`.

In any case, the reason your tests are failing right now is that you're missing a local dependency on lerna here, like in the PublishCommand/integration fixture:
```json
  "devDependencies": {
    "lerna": "__TEST_PKG_URL__"
  }
```